Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/broad-dsp-gcr-public/base/jre:11-debian
+FROM us.gcr.io/broad-dsp-gcr-public/base/jre:17-debian
```

Comment:
I know this predates this change, but why do we use the debian image here and temurin elsewhere?

Reply:
I think this is just naming conventions from `dsp-appsec-blessed-images`. The "-debian" image is built on top of Temurin: https://github.com/broadinstitute/dsp-appsec-blessed-images/blob/main/jre/Dockerfile.17-debian